org-ql-search.el

```diff
@@ -296,13 +296,16 @@ Valid parameters include:
   :ts-format  Optional format string used to format
               timestamp-based columns.
 
+  :not-links  Optional boolean.  If non-nil, don't make headings
+              into links.
+
```

Comment:
Let's avoid making a double-negative (e.g. as `:not-links nil` would be).  Instead, let's call it `:link` and have it default to `t`, so to disable it the user would use `:link nil`).

For the docstring, how about: "Link headings to their entries (default behavior)"